crates/oci/src/client.rs

```diff
@@ -106,7 +106,10 @@ impl Client {
                 // locked application file with the file digest.
                 for entry in WalkDir::new(&source) {
                     let entry = entry?;
-                    if entry.file_type().is_file() && !entry.file_type().is_dir() {
+                    if entry.file_type().is_file()
```

Comment:
We should at least include a DEBUG log here for skipped blobs to help future-us with debugging